data-sources/lighttpd/config.yml

```diff
@@ -0,0 +1,22 @@
+id: lighttpd
```

Comment:
Looks good..
There are few more fields which are to be added.

Request you to kindly follow this contribution guide
guide: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1
Thank you 😃